core/exceptions.py

```
@@ -0,0 +1,166 @@
+    pass
+
+
+class LLMTimeoutError(LLMError):
+    """LLM 超时"""
+    pass
+
+
+class LLMValidationError(LLMError):
+    """LLM 输出验证失败"""
+    
+    def __init__(self, message: str, response: str = None, schema: str = None):
```

Ruff (0.15.9)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)